src/Forms/__tests__/Dropdown.test.tsx

```diff
@@ -418,7 +419,7 @@ describe('Dropdown', function () {
       strictEqual(style.display, 'none');
     });
   });
-  context('when disabled prop is true', function () {
+  context('when disabled prop within an option is true', function () {
```

Comment:
This test (and the one below it) aren't named correctly. The prop is on the component, not the options.

Reply:
Before this PR, there was already a way to disable the option within the dropdown, so I had just named the test "when disabled prop is true." Now we can make the entire dropdown disabled in addition to having individual options disabled, so I'm not sure how to describe the tests? Should I say for line 422 "when disabled prop for an option is true"? For the new `disabled` prop for the overall dropdown, I described it "when disabled prop is true" (line 482 of the test). 
![image](https://user-images.githubusercontent.com/29589689/99423338-723e6d80-28ce-11eb-80dc-9261a7769f03.png)

Reply:
Oh right! Sorry. I didn't realize that you could also disable `<option>`s. I thought that was just an old holdover from the previous implementation :upside_down_face: .